### ISSUE TITLE
build files are wrong

### ISSUE THREAD

**Issue by JazzMaster** [Galleondragon/qb64#20]
the build files are wrong as the units are built on sin (windows). Line endings of all units and makefiles need to be corrected. especially the unix packages...need to be built on unix(or in a vm). This is a lot of work for those just wanting to "build and go". Cant speak if OSX and sin have this problem as well because im running ubu 17. you can subversion commit now to github, so just stay in the box when working on unix code unless you run it natively on unix.

This is a lot of hassle but it will eventually build after hacking the hell out of build files. I had to remove the /bin/bash references despite all of the dos2unix conversion. This isnt per-se needed but I kept getting ^M where they shouldnt have been despite conversion.Removing the line seems to work for now. Building in unix should get rid of this problem entirely.

 Also the configure file has two non utf8 chars in it that dont translate. removing them in the awk has no detrimental effect, rather curious why there there.

**Comment by FellippeHeitor** [comment]
If you download the zip straight from GitHub, you likely won't have issues with the setup script in Linux/macOS. The packages available from www.qb64.org also have this issue handled. The packages available from www.qb64.net still present the issue because the server where the auto build scripts are being run for qb64.net are Windows-based and it hasn't been possible to change that yet.

Aware of the issue, we've bundled instructions in the packages: https://raw.githubusercontent.com/Galleondragon/qb64/master/README%20for%20Linux%20and%20OS%20X.txt

In short, for Linux run this line from QB64's folder:
    `find . -name '*.sh' -exec sed -i "s/\r//g" {} \;`

and for macOS run:
    `find . -name '*.command' -exec perl -pi -e 's/\r\n|\n|\r/\n/g' {} \;`

That should suffice to allow building QB64 for *nix based systems.

Can you point the utf8 chars that don't translate in the scripts, please?
